### DISCUSSION TITLE
Graphql query - Order by related category title

### DISCUSSION THREAD

**Discussion by pnpaes** [craftcms/cms#10800] [Q&A] [answered]
Hi, I have an entry that has a few category fields.
When fetching data using Graphql, is it possible to achieve something like this `orderBy:"category_filed.title desc"` ?
I know that the example above will not work, but is there an alternative?

**Comment by brandonkelly** [comment, marked as answer]
This is not possible out of the box.

You could achieve it using the [Preparse Field](https://plugins.craftcms.com/preparse-field) plugin, though. With the plugin installed, create a new Preparse field that stores the related category’s title. Then add the field to your entry type’s field layout, and resave your entries using the `php craft resave/entries` command. Then you can order your entries based on the Preparse field’s handle.
